internal/router/middlewares/ratelim.go

```diff
@@ -59,50 +41,9 @@ func RateLimitController(cfg RateLimiterConfig) (mux.MiddlewareFunc, error) {
 	if err != nil {
 		return nil, fmt.Errorf("creating default rate limiter: %s", err)
 	}
-	customRLs := make(map[string]*httplimit.Middleware, len(cfg.JSONRPCMethodLimits))
-	for route, routeCfg := range cfg.JSONRPCMethodLimits {
```

Comment:
the RPC rate limiting is gone now, and this means we only need to use the default rate limiter.